data_pipeline/sentiment/dags/crypto/el_airflow_crypto_avalanche.py

```diff
@@ -0,0 +1,56 @@
+# imports important for Airflow
+import pendulum
+from airflow.decorators import dag, task
+import logging
+import time
+from newsapi import NewsApiClient
+from airflow.providers.mongo.hooks.mongo import MongoHook
+from datetime import datetime, timedelta
+from airflow.operators.trigger_dagrun import TriggerDagRunOperator
+@dag(
```

Comment:
This should be spaced after imports. Please add flake8 to your CI pipeline and ensure you use black to format before pushing.